### ISSUE TITLE
[FEATURE]: Ability to add a comment to each FQDN or or IP in a file 

### ISSUE THREAD

**Issue by miro444** [Matty9191/ssl-cert-check#125]
In some scenarios where split DNS is used internally, you want to be able to check both destinations. The only way to do this is to specify the target IP instead of the FQDN. Having multiple IPs then, it is hard to determine what device it is without doing internal IP search. 

Therefore, having the ability to add a "friendly name" or "description" to a target would solve this within the file. 
 
Example:

domain.com 443 # load balancer 1
anotherdomain.com 443 # web server xyz

**Comment by pothi** [comment]
Good idea. Currently, you can insert comments on their own line like this...

```
# load balancer 1
domain.com 443

# web server xyz
anotherdomain.com 443
```
